### DISCUSSION TITLE
Please update your VS Code error (Mac OSX Visual Studio code)

### DISCUSSION THREAD

**Discussion by restfulhead** [community/community#53708] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Copilot does not seem to work on my machine. The following error message appears: `GitHub Copilot requires VS Code 1.57+. Please update your VS Code`

I am using the latest, official VS Code binary distribution for Mac Apple Silicon:

```
Version: 1.77.3
Commit: 704ed70d4fd1c6bd6342c436f1ede30d1cff4710
Date: 2023-04-12T09:41:48.638Z (1 wk ago)
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Darwin arm64 22.4.0
Sandboxed: No
```

I saw an answer to a similar question proposing to add the following to `Configure runtime agents`: 

```
	"enable-proposed-api":["github.copilot"]
```

I did that and restarted VSCode. The same error occurs.

**Comment by duriann** [comment]
the same to me

**Comment by restfulhead** [comment, marked as answer]
After updating the extension to the latest version `v1.84.51` and restarting VSCode it is now working.
